docs/docs/building-modules/09-autocli.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
Could you bring in the changes from this file in this PR too: https://github.com/cosmos/cosmos-sdk/pull/13811/files#diff-ee2f066a7b97e94071ef696e92ced9fac24f3f3714a83fdf9da85732af1e8a2aR62